integration_tests/pages/page.ts

```diff
@@ -70,6 +70,14 @@ export default abstract class Page {
     cy.get('.govuk-back-link').should('have.attr', 'href', href)
   }
 
+  shouldContainLink(text: string, href: string): void {
+    cy.contains('.govuk-link', text).should('have.attr', 'href', href)
+  }
```

Comment:
Alphabetical?

Reply:
Alphabetical function parameters?

Reply:
`should` methods: https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/commit/55e2854d25b3d72abdea1072dccc1aa0b98eef7c / https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/145/commits/34bcbebd6502298ebba6e98a1caa29ab24e6808a

Reply:
I've updated for this PR, but this is probably something we should address with linting/auto-formatting, as it's easy to forget/miss those old commit messages. Do you think that's possible?